mnamer/setting_store.py

```diff
@@ -218,6 +218,16 @@ class SettingStore:
             help="--episode-format: set episode renaming format specification",
         )(),
     )
+    relocation_mode: Optional[Union[RelocateType, str]] = dataclasses.field(
+        default=RelocateType.DEFAULT.value,
+        metadata=SettingSpec(
+            dest="relocation_mode",
+            choices=[ix.value for ix in RelocateType],
+            flags=["--link"],
```

Comment:
Need a better option flag name than "link". One encompassing the range of operations this flag enables. It enables move, copy, hardlink, symlink and copying with metadata. I decided on '--relocate-operation' for now.